platform/ext/target/nordic_nrf/common/core/native_drivers/spu.c

```diff
@@ -126,27 +126,28 @@ void spu_regions_reset_unlocked_secure(void)
     for (size_t i = 0; i < NUM_FLASH_SECURE_ATTRIBUTION_REGIONS ; i++) {
```

Comment:
For spu_peripheral_config_secure/non_secure.

If we add:
SPU_SECURE_DMA_SECURE
SPU_SECURE_DMA_NONSECURE

Then we can add reduce these two functions to one spu_config_peripheral, add secure_attr and secure_dma as argument, and  rename periph_lock to lock_conf.

Not sure if worth it though, maybe just make it a common function to be used by those two functions instead?

Reply:
discussed offline. Will treat the peripheral API as out-of-scope.